src/style.scss

```diff
@@ -9,3 +9,81 @@
 
 @import './../node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
 @import './../node_modules/@alaskaairux/webcorestylesheets/dist/core';
+
+/* stylelint-disable selector-class-pattern, color-function-notation, number-leading-zero, number-max-precision, color-hex-case, color-hex-length, max-line-length, selector-max-class, max-nesting-depth, selector-max-combinators, selector-max-compound-selectors */
+
+:host([anchornav]) {
+  position: relative;
+
+  .label-container {
+    padding-bottom: var(--auro-size-200);
+    color: var(--auro-color-text-secondary-on-light);
+
+    &.hidden {
+      display: none;
+    }
+  }
+
+  @include auro_breakpoint($max: $auro-breakpoint-sm) {
+    #showHideBtn {
+      display: block;
+    }
+  }
+}
+
+:host([anchornav]:not([aria-expanded])) {
+  @include auro_breakpoint($max: $auro-breakpoint-sm) {
+    &:before {
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      display: block;
+      width: 100%;
+      height: 100%;
+      background: linear-gradient(180deg, rgba(255, 255, 255, 0.0001) 0%, rgba(255, 255, 255, 0.00874633) 6.67%, rgba(255, 255, 255, 0.0356065) 13.33%, rgba(255, 255, 255, 0.0817517) 20%, rgba(255, 255, 255, 0.147497) 26.67%, rgba(255, 255, 255, 0.231852) 33.33%, rgba(255, 255, 255, 0.331951) 40%, rgba(255, 255, 255, 0.442747) 46.67%, rgba(255, 255, 255, 0.557353) 53.33%, rgba(255, 255, 255, 0.668149) 60%, rgba(255, 255, 255, 0.768248) 66.67%, rgba(255, 255, 255, 0.852603) 73.33%, rgba(255, 255, 255, 0.918348) 80%, rgba(255, 255, 255, 0.964494) 86.67%, rgba(255, 255, 255, 0.991354) 93.33%, #FFFFFF 100%);
+      content: '';
+      pointer-events: none;
+    }
+
+    #showHideBtn {
+      [more] {
+        display: none;
+      }
+
+      [less] {
+        display: inline;
+      }
+    }
+
+    ::slotted(auro-anchorlink) {
+      display: none;
+    }
+
+    ::slotted(auro-anchorlink:nth-of-type(1)),
+    ::slotted(auro-anchorlink:nth-of-type(2)),
+    ::slotted(auro-anchorlink:nth-of-type(3)) {
```

Comment:
Does this create a limitation?

Reply:
It does create a limitation in the fact that only the first three links are displayed in the mobile version, but this is following the design spec.